<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="boxInput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>511</width>
      <height>551</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonHog">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>670</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>hog</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLoad">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>600</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>loadImage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonHisto">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>570</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Grayscale Histogram Level</string>
    </property>
   </widget>
   <widget class="QLabel" name="boxOutput">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>541</width>
      <height>541</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPrediction">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>720</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Alphabet Prediction</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCanny">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>620</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Canny</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1260</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
